src/System.Diagnostics.DiagnosticSource/tests/HttpHandlerDiagnosticListenerTests.cs

```diff
@@ -149,6 +149,7 @@ public void TestReflectInitializationViaSubscription3()
         /// Test request cancellation: reflection hook does not throw
         /// </summary>
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "dotnet/corefx #19286")]
```

Comment:
This tests seems to be fixed by @lmolkova but I'm not sure as we didn't get a response from her when we asked on the issue. 

So I don't know if we should disable this test. Also there is two issues tracking this test, so could you please update the issue # to 19280?

@danmosemsft should we disable this test? or wait until we get a response from @lmolkova?